amqphandler/amqphandler_test.go

```diff
@@ -320,62 +329,250 @@ func TestSendMessages(t *testing.T) {
 	}
 }
 
-func TestReceiveMessages(t *testing.T) {
-	systemID := "testID"
+func TestDesiredStatusMessages(t *testing.T) {
```

Comment:
Add into TestSendMessages as the dedicated item.

Reply:
decide to use another approach to test desired message